test/cpp/jit/test_save_load.cpp

```diff
@@ -35,7 +35,7 @@ template <class Functor>
 inline void expectThrowsEq(Functor&& functor, const char* expectedMessage) {
   try {
     std::forward<Functor>(functor)();
-  } catch (const Error& e) {
+  } catch (const c10::Error& e) {
```

Comment:
it's not crystal clear to me why this PR causes the c++ compiler to use torch::autograd::Error here